cmd/re/main.go

```diff
@@ -149,87 +122,79 @@ func main() {
 	}()
 	tracer := tp.Tracer(svcName)
 
-	rePubSub, err := nats.NewPubSub(ctx, cfg.BrokerURL, logger)
+	httpServerConfig := server.Config{Port: defSvcHTTPPort}
+	if err := env.ParseWithOptions(&httpServerConfig, env.Options{Prefix: envPrefixHTTP}); err != nil {
+		logger.Error(fmt.Sprintf("failed to load %s HTTP server configuration : %s", svcName, err))
+		exitCode = 1
+
+		return
+	}
+
+	rePubSub, err := smqbrokers.NewPubSub(ctx, cfg.BrokerURL, logger)
```

Comment:
What do you think about renaming `rePubSub` to something like `msgSubscriber` or `msgSub`?
These names could better reflect the actual responsibility of the component and make the intent clearer to us.